desktop-widgets/subsurfacewebservices.cpp

```diff
@@ -353,11 +356,11 @@ void WebServices::updateProgress(qint64 current, qint64 total)
 
 void WebServices::connectSignalsForDownload(QNetworkReply *reply)
 {
-	connect(reply, SIGNAL(finished()), this, SLOT(downloadFinished()));
-	connect(reply, SIGNAL(error(QNetworkReply::NetworkError)),
-		this, SLOT(downloadError(QNetworkReply::NetworkError)));
-	connect(reply, SIGNAL(downloadProgress(qint64, qint64)), this,
-		SLOT(updateProgress(qint64, qint64)));
+// 	connect(reply, &QNetworkReply::finished, this, &WebServices::downloadFinished); TODO: wrong
```

Comment:
And these connects wont work either because there are no such slots in WebServices.

Reply:
As far as I know, the webservice thing was discontinued. In any case, thanks for all the cleanups / fixes!

Reply:
Actually, perhaps this did work before your patch:

note how `DivelogsDeWebServices` does have a `DownloadFinished` slot. Not sure how Qt handles that case with the string-signature thing.

Easy fix would be adding a pure virtual function. 

But I would prefer merging these two classes - what is the point of a parent class when there is a single subclass...?